src/black/strings.py

```diff
@@ -4,8 +4,9 @@
 
 import re
 import sys
+import unicodedata
 from functools import lru_cache
-from typing import List, Pattern
+from typing import List, Pattern, Text
```

Comment:
Text is only relevant for Python 2/3 compatibility. Black is Python 3-only, so just use `str`.

Reply:
Instead of importing `typing.Text`, I replaced `Text` with `str`!